web/react-web/src/Content/Graph/Graph.js

```diff
@@ -142,9 +159,32 @@ class Graph extends Component {
                 name = `Active cases for ${one.long_name}`;
             }
 
+            let yRatio = 1.0;
+            switch (dataScaling) {
+                case SCALING_SAME_PEAK:
+                    let maxValue = one.max_value;
+                    if (one.type !== 'prediction') {
+                        maxValue = Math.max(...one.values);
```

Comment:
`one.values` are ative cases or predictions?

Reply:
Both, it just depends on type. For predictions, we have maxValue precalculated. For active cases, we don't. I was already thinking on precalculation of this value for active cases on python side, but it would be another PR with greater set of optimizations.